LICENSE_HEADER.txt

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2023-present Contributors to the Packit Project.
+
+SPDX-License-Identifier: MIT
```

Comment:
why do we use a different license header here than for other repos?

Reply:
It was recently bootstrapped and I followed the SPDX specification that allows the copyright to be easily parsed (same as with the license identifier), though even in their own examples they also have the headers we have in other projects (license identifier machine-readable with copyright nearby (not prefixed with the `SPDX-FileCopyrightText: `)), so it didn't make much sense to me to convert everything to this, but it feels like a nice-to-do thing with newer stuff